tests/component/test_export_signal_properties.py

```diff
@@ -0,0 +1,81 @@
+"""Contains a collection of pytest tests that validates export signal properties."""
+
+import pytest
+
+import nidaqmx
+from nidaqmx.constants import ExportAction
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.system.device import Device
+
+
+def test__export_signal__get_enum_property__returns_value(any_x_series_device: Device):
+    """Test to validate getter for export signal property of enum type."""
+    with nidaqmx.Task() as task:
```

Comment:
#241 added a fixture for channels, which I like. There is one issue with their impl that I believe is resolvable, but I think its a good idea to do the same for these tests to remove a bunch of duplicate code!

Reply:
Added fixtures for this block of code.